similarity/pom.xml

```diff
@@ -16,12 +16,61 @@
             <artifactId>illinois-core-utilities</artifactId>
             <version>3.1.7</version>
```

Comment:
Whatever ide you use, could you format this, because this is clearly not formatted.